terraform/aws/aws.tf

```diff
@@ -2,7 +2,8 @@ variable "availability_zone" {}
 variable "control_count" {default = "3"}
 variable "control_type" {default = "m1.small"}
 variable "datacenter" {default = "aws"}
-variable "data_volume_size" {default = "100"} # size is in gigabytes
+variable "control_data_volume_size" {default = "20"} # size is in gigabytes
+variable "resource_data_volume_size" {default = "100"} # size is in gigabytes
```

Comment:
Yes, we will.
https://github.com/CiscoCloud/microservices-infrastructure/issues/809